homeassistant/components/tuya/strings.json

```diff
@@ -523,6 +523,9 @@
       "phase_c_voltage": {
         "name": "Phase C voltage"
       },
+      "add_ele": {
+        "name": "Add energy"
```

Comment:
This is somewhat unclear. Is it a button to press to "add energy"? (Add is a verb/action) What is the meaning of the dp code?

Reply:
@elupus the latest commit fix this issue, it's "Current energy level", not "Add energy", sorry for the confusion.